app/views/state_file/questions/az_review/edit.html.erb

```diff
@@ -45,8 +45,14 @@
   <% if current_intake.ask_whether_incarcerated? %>
     <div id="incarcerated" class="blue-group">
       <div class="spacing-below-5">
-        <p class="text--bold spacing-below-5"><%=t(".excise_credit") %></p>
+        <p class="text--bold spacing-below-5"><%=t(".was_incarcerated") %></p>
```

Comment:
I did not find a key `state_file.questions.az_review.edit.was_incarcerated` - Did I miss it?